plugins/vst_base/RemoteVstPlugin32.cmake

```diff
@@ -1,14 +1,24 @@
+# INSTALL_EXTERNAL_PROJECT: install a project created with ExternalProject_Add in the
+# parent project's install time.
+#
+# Description:
+# In a regular scenario, cmake will install external projects
+# BEFORE actually building the parent project. Since the building
+# process may use installed components from the project.
+# We want to give the external project the ability to install
+# files directly to the parent's install. Therefore, we have to
+# manually trigger the install stage with the parent's INSTALL_PREFIX.
+MACRO(INSTALL_EXTERNAL_PROJECT name)
+	ExternalProject_Get_Property(${name} BINARY_DIR)
+
+	install(CODE
+			"execute_process(COMMAND ${CMAKE_COMMAND} \"-DCMAKE_INSTALL_PREFIX=\${CMAKE_INSTALL_PREFIX}\" -P cmake_install.cmake
```

Comment:
According to the [`CMake documentation`](https://cmake.org/cmake/help/latest/command/install.html#generated-installation-script), it seems like we should pass `COMPONENT` and `BUILD_TYPE` as well.

Reply:
it seems to have a syntax problem. I don't know why.

Reply:
It seems like we can just include the target's `cmake_install.cmake` as well.